src/rust/catloop/mod.rs

```diff
@@ -254,95 +256,102 @@ impl CatloopLibOS {
         let mut qtable: RefMut<IoQueueTable<CatloopQueue>> = self.qtable.borrow_mut();
 
         // Issue accept operation.
-        match qtable.get(&qd) {
+        let (control_duplex_pipe, local): (Rc<DuplexPipe>, SocketAddrV4) = match qtable.get_mut(&qd) {
```

Comment:
Did you have any motivation for moving out from the previous match construct?

Reply:
Yes, I need to borrow the queue again to insert the pending op on line 322, and I can't do that within the same scope as line 261.

Reply:
Sure.